edm4eic.yaml

```diff
@@ -244,6 +244,22 @@ datatypes:
   ## ==========================================================================
   ## Calorimetry
   ## ==========================================================================
+  edm4eic::SimCalorimeterPulse:
+    Description: "Simulated calorimeter pulse prior to digitization."
+    Author: "D. Anderson, S. Joosten"
+    Members:
+      - uint64_t            cellID          // ID of the readout cell for this pulse.
+      - float               energy          // Total energy for this pulse in [GeV].
+      - float               position        // Position of the hit in world coordinates [mm].
```

Comment:
(comment from @simonge and @ruse-traveler )
Should be a Vector3f.

Reply:
Fixed